gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -259,6 +259,7 @@ public void orchestrate(Spec spec) throws Exception {
         // In this case, the current time is used as the flow executionId.
         flowMetadata.putIfAbsent(TimingEvent.FlowEventConstants.FLOW_EXECUTION_ID_FIELD,
             Long.toString(System.currentTimeMillis()));
+        flowMetadata.put(TimingEvent.METADATA_MESSAGE, "Flow compilation failed");
```

Comment:
Can we add spec.getCompilationErrors() also? We will know which execution failed with which config exception.

Reply:
We could, the reason I didn't want to is it would be kind of misleading right now since sometimes that will have unrelated missing config messages. I guess for now I could just add it and say "it may be due to these errors".

Reply:
Right.

Reply:
So the next challenge for us is to find failures in resolving which flow/job template are the correct errors.